backend/translations/validators/validators.fr.yaml

```diff
@@ -0,0 +1,9 @@
+user:
```

Comment:
Missing header with license , can be fixed on a next PR (@eddsaura )

Reply:
Yes, once we push the new files I'll fix the headers. 👍🏽  Thanks for pointing out! 😄